server/src/gql/resolver.py

```diff
@@ -8,7 +8,8 @@
 from src.gql.graphql import NewMatesInput
 from src.model.family import Family
 from src.model.mate import Mate
-from src.model.user import UNASSIGNED_FAMILY_INTERVAL, User
+from src.model.user import User
+from src.types.new_family import UNASSIGNED_FAMILY
```

Comment:
moved these types out to another file to have better separation of concerns as well as to avoid circular dependencies.